chia/rpc/full_node_rpc_api.py

```diff
@@ -789,13 +789,15 @@ async def get_fee_estimate(self, request: Dict) -> Dict[str, Any]:
         mempool_max_size = estimator.mempool_max_size()
         blockchain_state = await self.get_blockchain_state({})
         synced = blockchain_state["blockchain_state"]["sync"]["synced"]
-        peak_height = blockchain_state["blockchain_state"]["peak"].height
-        last_peak_timestamp = blockchain_state["blockchain_state"]["peak"].timestamp
+        peak = blockchain_state["blockchain_state"]["peak"]
+        peak_height = peak.height if peak else uint32(0)
+        last_peak_timestamp = peak.timestamp if peak else uint64(0)
         peak_with_timestamp = peak_height
-        while last_peak_timestamp is None:
-            peak_with_timestamp -= 1
-            block_record = self.service.blockchain.height_to_block_record(peak_with_timestamp)
-            last_peak_timestamp = block_record.timestamp
+        if peak:
+            while last_peak_timestamp is None:
+                peak_with_timestamp -= 1
+                block_record = self.service.blockchain.height_to_block_record(peak_with_timestamp)
+                last_peak_timestamp = block_record.timestamp
```

Comment:
How would you feel about this?  Seems a bit more clear as to the handling of the two cases.

```suggestion
        if peak is None:
            peak_height = uint32(0)
            last_peak_timestamp = uint64(0)
        else:
            peak_height = peak.height
            last_peak_timestamp = peak.timestamp
            peak_with_timestamp = peak_height
            while last_peak_timestamp is None:
                peak_with_timestamp -= 1
                block_record = self.service.blockchain.height_to_block_record(peak_with_timestamp)
                last_peak_timestamp = block_record.timestamp
```

Reply:
Yes, looks clearer to me.

Reply:
@altendky Something is preventing me from using the github `Commit suggestion` button - I just pushed these changes up from local.

Reply:
Yeah, I wasn't sure if it would work, sorry.  I cheated since it wouldn't let me create it over a deleted line.